.gitignore

```diff
@@ -4,3 +4,5 @@ vsql-server
 *.vsql
 .vscode
 .vlang_tmp_build/
+
+scripts/generate-snippets
```

Comment:
Was this just left over?

Reply:
Apparently .vsh generates this binary.  I think we should ignore it.